### DISCUSSION TITLE
How to list sub commands in runtime

### DISCUSSION THREAD

**Discussion by horihiro** [denoland/deno#10100] [Q&A]
Does `deno` have a way to list the sub-commands supported in runtime?

In the following repo, I'm trying to use `deno compile` in the CLI tool running on `deno` in order to reduce the size of the deployment package.
https://github.com/horihiro/azure-functions-deno-worker/blob/bundleAsExecutable/denofunc.ts#L97

I want to check if the `deno` which a developer uses is supporting the sub-command `compile` at the beginning of the CLI tool.

I would appreciate any insights.

**Comment by jsejcksn** [comment]
@horihiro See the code in the script (`parse_subcommands_from_help.ts`) that I posted in this comment:
https://github.com/hayd/deno-docker/issues/119#issue-851876807
